bedrock/mozorg/templates/mozorg/about/governance/organizations.html

```diff
@@ -14,18 +14,20 @@ <h1 class="title-shadow-box">{{_('Mozilla Organizations')}} </h1>
 
 <p>{{_('There are several organizations that support the Mozilla community and ')}}<a href="{{ url('mozorg.about.manifesto') }}">{{_('Mozilla’s principles.</a> They include the non-profit Mozilla Foundation as well as the wholly owned taxable subsidiary, the Mozilla Corporation. Mozilla considers itself a <a href="http://commonspace.wordpress.com/category/hybrid/">hybrid organization</a>, combining non-profit and market strategies to ensure the Internet remains a shared public resource.')}}</p>
 
-<p>{{_('This page provides information about the various organizations and their relationships to each other. If you have any questions about this, please feel free to <a href="//www.mozilla.org/contact">contact us</a>')}}</p>
+<p>{{_('This page provides information about the various organizations and their relationships to each other. If you have any questions about this, please feel free to ')}}<a href="{{ url('mozorg.contact.contact-landing') }}">{{_('contact us</a>.')}}</p>
```

Comment:
I don't think this page is actively localized, but we should avoid splitting a single sentence into multiple strings like this. I would follow what you have done below for this paragraph as well.